server/app/routes.js

```diff
@@ -0,0 +1,44 @@
+import AuthController from './controllers/AuthController';
```

Comment:
What about breaking this file into separate files for each controller to manage. So you can have a user-route file and you import the userController there